config/defaults.conf

```diff
@@ -87,8 +87,8 @@ reload_indexing = 0  // If true, force the rebuilding of the index files in prep
                      // exist.
 target_train_data_fraction = 1  // Use only the specified fraction of the training data in the do_target_task_training phase.
                         // should not impact pretraining, even for the same task.
-                        // pretrain_data_fraction should not be used together with                 // target_train_data_fraction
-                        // both should not be less then 1 at a same time
+                        // pretrain_data_fraction can be used together with target_train_data_fraction. Both the fractions can be less then 1 at a same time.
+                        // The data is sorted and serialized.  For e.g., if both are set to 0.1, the training set will be same for both.
```

Comment:
Say more—this might give the impression that it's sorted by text or by length, which would be a serious problem.